File: app/src/main/java/com/alvinhkh/buseta/kmb/model/network/KmbAnnounceRes.java
package com.alvinhkh.buseta.kmb.model.network;


import com.alvinhkh.buseta.kmb.model.KmbAnnounce;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class KmbAnnounceRes {

    @SerializedName("data")
    public ArrayList<KmbAnnounce> data;

    @SerializedName("result")
    public Boolean result;

    public String toString() {
        return "KmbAnnounceRes{data=" + this.data + ", result=" + this.result + '}';
    }

}

File: app/src/main/java/com/alvinhkh/buseta/kmb/model/network/KmbAnnounceRes.kt
package com.alvinhkh.buseta.kmb.model.network


import com.alvinhkh.buseta.kmb.model.KmbAnnounce
import com.google.gson.annotations.SerializedName

import java.util.ArrayList

data class KmbAnnounceRes(
        @SerializedName("data")
        var data: ArrayList<KmbAnnounce> = arrayListOf(),

        @SerializedName("result")
        var result: Boolean = false
)